docs/intro/maps.rst

```diff
@@ -491,20 +491,50 @@ The meta information informs `sunpy.map.Map` of the correct coordinate informati
 See this :ref:`sphx_glr_generated_gallery_map_map_from_numpy_array.py` for a brief demonstration of generating a Map from a data array.
 
 The keys required for the header information follow the `FITS standard <https://fits.gsfc.nasa.gov/fits_dictionary.html>`__.
-**sunpy** provides a Map header helper function to assist in creating a header that contains the correct meta information.
-This includes a :func:`~sunpy.map.header_helper.meta_keywords` function that will return a `dict` of the meta keywords used when creating a Map.
-
-.. code-block:: python
-
-    >>> from sunpy.map.header_helper import meta_keywords
-
-    >>> meta_keywords() # doctest: +SKIP
-    {'cunit1': 'Units of the coordinate increments along naxis1 e.g. arcsec **required',
-     'cunit2': 'Units of the coordinate increments along naxis2 e.g. arcsec **required',
-     'crval1': 'Coordinate value at reference point on naxis1 **required'
-     ...
-
-The utility function :func:`~sunpy.map.header_helper.make_fitswcs_header` will return a header with the appropriate FITS keywords once the Map data array and an `astropy.coordinates.SkyCoord` or `sunpy.coordinates.frames` is provided.
+All the metadata keywords that a Map will parse along with their description are listed below:
+
+cunit1: Units of the coordinate increments along naxis1 e.g. arcsec (required)
```

Comment:
This currently looks like: https://sunpy--6743.org.readthedocs.build/en/6743/intro/maps.html#creating-custom-maps

Hopefully turning it into a table will look nicer.

Reply:
i see, totally, will get on that :+1:

Reply:
I think this should probably go at the end of the page as well.